QUEST.tsv

```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
+QUEST_20150428_001958	$(The book has rotted so much that nothing inside can be read.)
+QUEST_20150428_001959	$The minister said, {nl}"We already heard the words of His Majesty. {nl}We don't know exactly why, but the reason this royal tomb was built was not to show off, {nl}but to prepare for the forecast that may bring chaos to the kingdom and change the future of the humans. {nl}If we don't do this important task, who will?
+QUEST_20150428_001960	$Warning Sign for Unauthorized Entrants
+QUEST_20150428_001961	$To protect the sacred secret of Zachariel King, no one may enter this facility.
+QUEST_20150428_001962	$Anyone entering here unauthorized will be punished for their sin.
+QUEST_20150428_001963	$The Order to Move Out
+QUEST_20150428_001964	$This facility will soon be shut down. {nl}I ask all construction workers and related people of the royal tomb to move out.
+QUEST_20150428_001965	$The Diary of Baron Soldier, Chagos
+QUEST_20150428_001966	$The magician, who appeared from somewhere, told the ignorant baron that the object is located here.
+QUEST_20150428_001967	$The way to eternal youth. {nl}I can't find the way there: the place where only death awaits.
+QUEST_20150428_001968	$The Crumbling Document
+QUEST_20150428_001969	$The demon lord, Helgasercle, divided the spirit of the demon lord, Haubuck. {nl}This was used to power the barrier that now surrounds the Mage Tower and then was scattered throughout the kingdom.
+QUEST_20150428_001970	$Many magicians came to break the barrier, but they could not succeed with just the foundations stones that were gathered at the tower.
+QUEST_20150428_001971	$To all adventurers of the kingdom, {nl}the Mage Tower offers gold to those who bring back the pieces of Haubuck's spirit.
+QUEST_20150428_001972	$The Order to Defeat the Demons
+QUEST_20150428_001973	$I hereby order in the name of Vakarine! {nl}After Medis Diena, they can no longer be forgiven. {nl}They are trying to expand their power by penetrating into the crack that was created in the barrier.
+QUEST_20150428_001974	$All the Cupols here shall defeat demon lords Boolut, Nuaelle, and Baltrus first. {nl}The enemy will panic without their commanders and will be easier to defeat.
+QUEST_20150428_001975	$Cupols who can't attack should concentrate on blocking the crack. {nl}We can't allow more demons from going to the demon lords.
+QUEST_20150428_001976	$How should I live from now on? {nl}I should have moved to Klaipeda.
+QUEST_20150428_001977	$Since you've placed them into the inventory, you can read them anytime.
+QUEST_20150428_001978	$The power of Mirtis was so overwhelming that Cunningham had to make an important decision.
+QUEST_20150428_001979	$I hear the more and more monsters are appearing in the Baron Arleruno's land. {nl}Please defeat those monsters before they take any innocent lives.
```

Comment:
Should be "I hear **that** more and more monsters are appearing in the Baron Arleruno's land. "